providers/azureblob/src/main/java/org/jclouds/azure/storage/filters/SharedKeyLiteAuthentication.java

```diff
@@ -72,7 +74,11 @@
    private final Supplier<Credentials> creds;
    private final Provider<String> timeStampProvider;
    private final Crypto crypto;
+   private final String credential;
+   private final String identity;
    private final HttpUtils utils;
+   private final URI storageUrl;
+   private boolean sasAuthentication;
```

Comment:
Make this final once it is initialized int he constructor.

Reply:
Done!